capstone/src/main/webapp/js/servletConnection.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * TODO: NOT DONE (just a sketch)
+ * Gets suggestions to display to the users
+ * @return{void}
+ */
+function getSuggestions() {
+  fetch('/search').then((response) => response.json()).then((suggestions) => {
+    provideSuggestions(suggestions);
+  });
+}
+
+/**
+ * Sends final college list to servlet
+ * @return{void}
+ */
+function sendCollegeInformation() {
+  const params = new URLSearchParams();
+  const listItems = document.querySelector('ul').children;
+  const listArray = Array.from(listItems);
+  const index = 0;
+  for (elt in listArray) {
```

Comment:
```suggestion
  for (const elt of listArray) {
```
per [style guide](https://google.github.io/styleguide/jsguide.html#features-for-loops)